sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -110,6 +110,26 @@ static TestEnvironment()
         /// </summary>
         public string ClientSecret => GetVariable("CLIENT_SECRET");
 
+        /// <summary>
+        ///   The authority host of the cloud to use during Live tests. Recorded.
+        /// </summary>
+        public string AuthorityHost => GetRecordedVariable("AZURE_AUTHORITY_HOST");
```

Comment:
@pakrym:  You may want to give these a look.   I'm not sure whether or not some of these are a clean fit for Core.

Reply:
I move these environment variables to EventHubTestEnvironment.cs. Because Now these environment variables only be used in event hub test

Reply:
They'll also be needed in `Azure.Messaging.ServiceBus` so it is a candidate for centralization.   I'd like Pavel's opinion before we make any final decisions.

Reply:
Only variables set in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/common/TestResources/New-TestResources.ps1#L295-L301 should be available in the shared `TestEnvironment`, otherwise it's very hard to diagnose why these particular variables are not available when others always are.

Both AuthorityHost, ServiceManagementUrl and ResourceManager seem like great candidates for centralization but we should add them to the script `TestResources.ps1` first.